tests/e2e/configurer/upgrade.go

```diff
@@ -415,3 +402,32 @@ func (uc *UpgradeConfigurer) upgradeContainers(chainConfig *chain.Config, propHe
 	uc.t.Logf("upgrade successful on chain %s", chainConfig.Id)
 	return nil
 }
+
+// START: CAN REMOVE POST v17 UPGRADE
```

Comment:
```suggestion
// TODO: REMOVE POST v17 UPGRADE
```

Reply:
I like the START and END comments so when v17 is launched we should be able to just search these and know exactly which parts need to get removed (some portions are entire functions but others are portions within a function)

Reply:
okay that works too